packages/rs-drive-abci/src/execution/mod.rs

```diff
@@ -1,24 +1,10 @@
 /// Check tx module
 mod check_tx;
-/// Data triggers
-pub mod data_trigger;
 /// Engine module
 pub mod engine;
-/// Fee pools module
-pub mod fee_pools;
-/// Helper methods
-pub mod helpers;
-/// Initialization
-pub mod initialization;
-/// Masternode Identities
-mod masternode_identities;
-/// Processor module
-pub mod processor;
-/// Protocol upgrade
-pub mod protocol_upgrade;
+/// platform execution events
+pub(in crate::execution) mod platform_events;
```

Comment:
Very confusing name. It's not actually events just functions that implement execution logic.